.gitmodules

```diff
@@ -9,7 +9,7 @@
 [submodule "external/Java.Interop"]
     path = external/Java.Interop
     url = https://github.com/xamarin/java.interop.git
-    branch = main
+    branch = jonp-jdk-17-support
```

Comment:
Can we use java.interop/main before merging this? Or maybe this branch name is just leftover?

Reply:
Yes, I'll want to merge https://github.com/xamarin/java.interop/pull/1141 and update this to use xamarin/java.interop/main.  I forgot about that before making this a "normal" PR.  Doh!

Reply:
Updated, now using java.interop/main!